server/src/main/java/org/spine3/server/entity/RecordBasedRepository.java

```diff
@@ -284,9 +287,12 @@ protected E findOrCreate(I id) {
     /**
      * Converts the passed entity into the record.
      */
-    protected EntityRecord toRecord(E entity) {
+    protected EntityRecordWithColumns toRecord(E entity) {
         final EntityRecord entityRecord = entityConverter().convert(entity);
-        return entityRecord;
+        final Map<String, Column.MemoizedValue<?>> storageFields = Columns.from(entity);
```

Comment:
I don't like exposing `Column.MemoizedValue` out of its package.

Can we create a factory method in the  `EntityWithColumns`, which produces `EntityRecordWithColumns` instance out of `EntityRecord`?